_mo.scss

```diff
@@ -111,15 +109,15 @@
 }
 
 @mixin ellipsis($width: false) {
-  @if $width { width: $width };
   overflow:       hidden;
-  white-space:    nowrap;
   text-overflow:  ellipsis;
+  white-space:    nowrap;
```

Comment:
@ryancoughlin looks like the order stuff you changed here causes a few tests to fail should be a snap to fixup let me know if you are stuckkk this is 95% of the way there